app/repositories/page-content.tsx

```diff
@@ -1,12 +1,13 @@
 import { SectionHeader } from '../../components/SectionHeader'
 import { RepositoriesTable } from '@/repositories/RepositoriesTable'
-import { useSearchParams } from 'next/navigation'
```

Comment:
Changes in this file are unrelated to the PR but I was working around this area so added these on. 

1. show loading state on repo list table
2. use repo moderation states to show subject overview and subject status icon just like we do in the queue page instead of having it show just the handle in the first column and subject status in the last column.
3. add quick action panel in repositories page so that mods can directly action accounts from the page without having to go to profile screen first.
3. improve moderation status loading performance by bulk fetching using getRepos instead of calling getRepo on every did.